Description: Disable pytest_asyncio

Debian currently has incompatible versions of pytest_asyncio and
pytest, causing these tests to error on collection.

Author: Rebecca N. Palmer <rebecca_palmer@zoho.com>
Bug-Debian: https://bugs.debian.org/969050
Forwarded: not-needed https://github.com/pandas-dev/pandas/pull/35757

--- pandas-1.0.5+dfsg.orig/pandas/util/_test_decorators.py
+++ pandas-1.0.5+dfsg/pandas/util/_test_decorators.py
@@ -255,10 +255,6 @@ def check_file_leaks(func) -> Callable:
 
 
 def async_mark():
-    try:
-        import_optional_dependency("pytest_asyncio")
-        async_mark = pytest.mark.asyncio
-    except ImportError:
-        async_mark = pytest.mark.skip(reason="Missing dependency pytest-asyncio")
+    async_mark = pytest.mark.skip(reason="https://bugs.debian.org/969050")
 
     return async_mark
